llvm/include/llvm/ProfileData/InstrProfWriter.h

```diff
@@ -75,11 +75,16 @@ class InstrProfWriter {
   // deployment of newer versions of llvm-profdata.
   bool WritePrevVersion = false;
 
+  // The version we should use to write MemProf in.
```

Comment:
Maybe "The MemProf version we should write"? (the "write MemProf in" at the end is a little hard to parse)